dll/shellext/zipfldr/CZipFolder.hpp

```diff
@@ -380,7 +380,6 @@ class CZipFolder :
             ici.lpVerb = MAKEINTRESOURCEA(wParam);
             return spContextMenu->InvokeCommand(&ici);
         }
-        case DFM_INVOKECOMMANDEX:
```

Comment:
I do believe I put this here because without it it wouldn't work in Win7 or Win10,
but that might be because something else was unimplemented at the time...

Reply:
That can't be right. ROS has never supported it and on Windows this was wrong because you can't ask Windows to perform this command for you (because it's not a standard DFM_CMD_).

Anyway, I have found the original source of this "Windows 7 requirement" claim and it was not you that originally did it nor do I believe it's required. Upcoming PR...